vm/src/builtins/bytearray.rs

```diff
@@ -21,7 +21,9 @@ use crate::pyobject::{
     BorrowValue, Either, IdProtocol, IntoPyObject, PyClassImpl, PyComparisonValue, PyContext,
     PyIterable, PyObjectRef, PyRef, PyResult, PyValue, TypeProtocol,
 };
-use crate::sliceable::{PySliceableSequence, PySliceableSequenceMut, SequenceIndex};
+use crate::sliceable::{
+    PySliceableSequence, PySliceableSequenceMut, PySliceableSequenceOwner, SequenceIndex,
```

Comment:
I wonder if we could just import `PyClassDef` and instead of `Self::OWNER_TYPE` just use the `Self::NAME` that's already defined on `PyClassDef`

Reply:
If that's the case, I probably should redo this patch, remove the trait and only fix the tests.

Reply:
Yeah, I think that might be best. Sorry for leading you on a sort of wild goose chase :sweat_smile: